color_channel_swapper.py

```
@@ -0,0 +1,95 @@
+        self.save_btn = tk.Button(self.root, text="Save Image", command=self.save_image, state=tk.DISABLED)
+        self.save_btn.pack(pady=5)
+
+        self.img_label = tk.Label(self.root)
+        self.img_label.pack(pady=5)
+
+        # Add traces to channel1 and channel2
+        self.channel1.trace_add('write', self.update_swap_btn_state)
+        self.channel2.trace_add('write', self.update_swap_btn_state)
+
+    def load_image(self):
+        file_path = filedialog.askopenfilename(filetypes=[("Image Files", "*.png;*.jpg;*.jpeg;*.bmp;*.gif")])
```

Pylint (3.3.7)

[convention] Line too long (109/100)

(C0301)